NGitLab/Models/MergeRequestCreate.cs

```diff
@@ -22,6 +22,10 @@ public class MergeRequestCreate
         [JsonPropertyName("assignee_ids")]
         public int[] AssigneeIds;
 
+        [DataMember(Name = "reviewer_ids")]
+        [JsonPropertyName("reviewer_ids")]
+        public int[] ReviewerIds;
```

Comment:
You need to add the new member to [PublicAPI.Unshipped.txt](https://github.com/ubisoft/NGitLab/blob/main/NGitLab/PublicAPI.Unshipped.txt) to fix the CI.
You can use the quick fix in Visual Studio

![image](https://user-images.githubusercontent.com/509220/154704771-26f6fc8b-4ce6-4190-8be0-251c3ecfeb5d.png)